transitracker/templates/layout.html

```diff
@@ -33,80 +33,16 @@
             background-color: rgba(62,62,62,.8);
             color: white;
         }
-        .appName{
+        .navbar-brand{
             font-size: 1.5rem;
-            margin: .5 rem;
+            margin: .5rem;
         }
-        .navbar-links ul{
-            margin: 0;
-            padding: 0;
-            display: flex;
-        }
-        .navbar-links li{
-            list-style: none;
-        }
-        .navbar-links li a{
-            text-decoration: none;
-            padding: 1rem;
-            display: block;
-        }
-        .navbar a{
-            color: white;
-        }
-
-        .navbar-links ul li:last-child{
-            margin-left: auto;
-        }
-        .toggle-button{
-            position: absolute;
-            top: .75 rem;
-            right: 1rem;
-            display: none;
-            flex-direction: column;
-            justify-content: space-between;
-            width: 31px;
-            height: 21px;
+        .navbar-dark .navbar-nav .nav-link{
+          color: white;
+          opacity: 1;
         }
-        .toggle-button .bar{
-            height: 3px;
-            width: 100%;
-            background-color: white;
-            border-radius: 10px;
-
-        }
-        @media (max-width: 700px){
-            .navbar{
-                width: 100%;
-            }
-
-            .toggle-button{
-                display: flex;
-            }
-            .navbar-links{
-                display: none;
-                width: 100%;
-            }
-            .navbar{
-                flex-direction: column;
-                align-items: flex-start;
-            }
-            .navbar-links ul{
-                width: 100%;
-                flex-direction: column;
-            }
-            .navbar-links li{
-                text-align: center;
-            }
-            .navbar-links li a{
-                padding: 1rem;
-            }
-            .navbar-links ul li:last-child{
-                margin:auto;
-            }
-            .navbar-links.active{
-                display: flex;
-
-            }
```

Comment:
Do we not need these to define behavior on smaller screens?

Reply:
No, the Bootstrap class I used has built-in scaling for the navbar. I used is navbar-expand-md which has scaling that seemed to kick in at 768px width or less when I was testing in Chrome. There are other sizing variants of that class that I can play with if needed.

Reply:
Ok great. I'll approve the pull